### DISCUSSION TITLE
8090 port filtered from local IP

### DISCUSSION THREAD

**Discussion by fabiodeagostinigithub** [doitandbedone/ispyagentdvr-docker#430] [General]
I can't understand why, but my installation is only reachable using online link.
the localip:8090 is unreachable.
nmap to localhost works, nmap with LAN-IP doesn't work.
I can't realize where is the problem.
Maybe docker iptables?
I stopped firewalld and iptables on host vm with results...

**Comment by ispysoftware** [comment]
whatever it is It's nothing to do with the connection to ispyconnect.com
